string group_name
string data_name
---
bool resp
